libethcore/Transaction.cpp

```diff
@@ -130,6 +133,14 @@ Address const& TransactionBase::sender() const
 	return m_sender;
 }
 
+SignatureStruct const& TransactionBase::signature() const
+{ 
+	if (!m_vrs)
```

Comment:
We could return empty `optional<SignatureStruct>`, but I don't thinks it's any better, the user still would need to check if the returned value is empty or not. And if he forgets he'll just get a crash instead of uncaught exception